lib/fhir_models/bootstrap/model.rb

```diff
@@ -37,20 +44,12 @@ def method_missing(method, *_args, &_block)
         end
         return nil
       elsif !@extension.nil? && !@extension.empty?
-        ext = @extension.select do |x|
-          name = x.url.tr('-', '_').split('/').last
-          anchor = name.split('#').last
-          (method.to_s == name || method.to_s == anchor)
-        end
+        ext = find_extension(@extension, method)
```

Comment:
Since we're updating `method_missing`, could you rename the `method` parameter to `method_name`? `method` is a method in Ruby, so using it as a variable name is not great.